File: android-twa/app/src/main/java/app/polycal/widgets/NewEventWidget.java
package app.polycal.widgets;

import app.polycal.R;

/** Home-screen New Event title bar (PC-479). */
public class NewEventWidget extends ComposeAppWidget {
    @Override
    String mode() {
        return WidgetUrls.MODE_EVENT;
    }

    @Override
    int layoutRes() {
        return R.layout.widget_new_event;
    }

    @Override
    int headerId() {
        return R.id.widget_header;
    }

    @Override
    int inputId() {
        return R.id.widget_input;
    }

    @Override
    int sendId() {
        return R.id.widget_send;
    }

    @Override
    int rootId() {
        return R.id.widget_root;
    }
}

File: android-widgets/app/src/main/java/app/polycal/widgets/NewEventWidget.kt
package app.polycal.widgets

/** Homescreen widget: one-line title → New Event composer. */
class NewEventWidget : ComposeAppWidget() {
    override val mode = WidgetUrls.MODE_EVENT
    override val layoutRes = R.layout.widget_new_event
    override val headerId = R.id.widget_header
    override val inputId = R.id.widget_input
    override val sendId = R.id.widget_send
    override val rootId = R.id.widget_root
}
